demo/src/app/components/+popover/demos/class/class.html

```diff
@@ -0,0 +1,5 @@
+<button type="button" class="btn btn-primary"
+        popover="Vivamus sagittis lacus vel augue laoreet rutrum faucibus." [containerClass]="'customClass'">
```

Comment:
don't use `[]` binding for string setters
`[containerClass]="'customClass'"`
should be
`containerClass="customClass"`